editor/src/clj/editor/settings.clj

```diff
@@ -122,6 +123,13 @@
      :default-section-name (when-let [category-name (:default-category meta-info)]
                              (section-title category-name (get-in meta-info [:categories category-name])))}))
 
+(defn get-setting-build-error [setting-value meta-setting label]
+  (if (and (some? setting-value)
+           (some? (:range meta-setting)))
+    (let [error (validation/prop-outside-range? (:range meta-setting) setting-value (string/join "." (:path meta-setting)))]
+      (if (some? error)
+        (g/->error nil label :fatal nil error)))))
```

Comment:
Not all errors we show in UI are build errors, but all build errors - the errors we wanna show in UI. 

e.g. deprecation error should be shown in UI, but shouldn't prevent building. 

(I didn't add `resource/exists?` into build errors, because they are already handled in another place)